target/scripts/helpers/log.sh

```diff
@@ -42,32 +42,22 @@ RESET=$(echo -ne   '\e[0m')
 # If the first argument is not set or invalid, an error
 # message is logged. Likewise when the second argument
 # is missing. Both failures will return with exit code '1'.
-function _log
-{
+function _log() {
   if [[ -z ${1+set} ]]
   then
-    echo "Call to '_log' is missing a valid log level" >&2
+    _log 'error' "Call to '_log' is missing a valid log level"
     return 1
   fi
 
   if [[ -z ${2+set} ]]
   then
-    echo "Call to '_log' is missing a message to log" >&2
+    _log 'error' "Call to '_log' is missing a message to log"
     return 1
   fi
 
-  local MESSAGE LEVEL_AS_INT LOG_LEVEL_FALLBACK
-  MESSAGE="${RESET}["
+  local LEVEL_AS_INT MESSAGE="${RESET}["
```

Comment:
No need to squeeze this into one line. I don't mind it when it's declaration only, but not a fan with assignment involved:

```suggestion
  local LEVEL_AS_INT
  local MESSAGE="${RESET}["
```